build/js/quicklink.js

```diff
@@ -1,49 +1,59 @@
-import { listen } from 'quicklink';
+import { listen, prefetch } from 'quicklink';
 
 window.addEventListener( 'load', () => {
 	const exportedOptions = window.quicklinkOptions || {};
 
-	const options = {};
+	const listenerOptions = {};
 
 	// el: Convert selector into element reference.
 	if ( 'string' === typeof exportedOptions.el && exportedOptions.el ) {
-		options.el = document.querySelector( exportedOptions.el );
-	}
-
-	// urls: Verify we don't get an empty array, as that would turn off quicklink.
-	if ( Array.isArray( exportedOptions.urls ) && 0 < exportedOptions.urls.length ) {
-		options.urls = exportedOptions.urls;
+		listenerOptions.el = document.querySelector( exportedOptions.el );
 	}
 
 	// timeout: Verify we actually get an int for milliseconds.
 	if ( 'number' === typeof exportedOptions.timeout ) {
-		options.timeout = exportedOptions.timeout;
+		listenerOptions.timeout = exportedOptions.timeout;
+	}
+
+	// limit: Verify we actually get an int.
+	if ( 'number' === typeof exportedOptions.limit ) {
```

Comment:
Maybe this isn't needed, but it would seem consistent with some of the other checks:

```suggestion
	if ( 'number' === typeof exportedOptions.limit && exportedOptions.limit > 0 ) {
```